app/controllers/concerns/symphony_cgi.rb

```diff
@@ -23,7 +24,8 @@ def request_url(script, cgi_params)
   end
 
   def query(cgi_params)
-    URI.encode_www_form(cgi_params).split('&').select { |param| param =~ /=./ }.join('&')
+    cgi_params.delete_if { |k, v| v.nil? }
```

Comment:
Need to look at what is going on here per https://github.com/sul-dlss/libsys-webforms/issues/612

Reply:
Fixed https://github.com/sul-dlss/libsys-webforms/issues/612